Hunting Queries/MultipleDataSources/DisabledAccIPSigninWithRareRiskyOps.yaml

```diff
@@ -0,0 +1,91 @@
+id: 9adbd1c3-a4be-44ef-ac2f-503fd25692ee
```

Comment:
I see you made this a hunting query still, any reason not to make it a detection?

Reply:
I honestly think this is ready to be a detection, but I just left it as a hunting query per the initial recommendation. Do you think it is good to move to detections?